cloudify_types/cloudify_types/component/utils.py

```diff
@@ -133,3 +134,21 @@ def blueprint_id_exists(client, blueprint_id):
 def deployment_id_exists(client, deployment_id):
     deployment = get_deployment_by_id(client, deployment_id)
     return True if deployment else False
+
+
+def should_upload_plugin(plugin_yaml_path, existing_plugins):
+    with open(plugin_yaml_path, 'r') as plugin_yaml_file:
+        plugin_yaml = yaml.load(plugin_yaml_file)
```

Comment:
unfortunate that we `yaml.load` here, do we have a util to use instead of yaml directly?

maybe use safe_load? remember that `yaml.load` can run (python) code

Reply:
so maybe we need to switch to safe load in all other places?